app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -33,7 +34,9 @@ define([
           self._updateCachedProfileImage(account.get('uid'), profileImage);
           return profileImage;
         }, function (err) {
-          self.logError(err);
+          if (err.errno !== ProfileErrors.ERRORS.UNAUTHORIZED.errno) {
```

Comment:
You should be able to do `if (ProfileErrors.is(err, 'UNAUTHORIZED')) {` and that'll be a bit cleaner.